salt/metalk8s/addons/grafana/deployed.sls

```diff
@@ -0,0 +1,7513 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+
+{%- from "metalk8s/registry/macro.sls" import build_image_name with context %}
+
+# The content below has been generated from
+# https://github.com/coreos/prometheus-operator, v0.24.0 tag,
+# with the following command:
+#   hack/concat-kubernetes-manifests.sh $(find contrib/kube-prometheus/manifests/ \
+#     -name "grafana-*.yaml") > deployed.sls
+# In the following, only container image registries have been replaced.
+
+---
+apiVersion: v1
+data:
+  prometheus.yaml: ewogICAgImFwaVZlcnNpb24iOiAxLAogICAgImRhdGFzb3VyY2VzIjogWwogICAgICAgIHsKICAgICAgICAgICAgImFjY2VzcyI6ICJwcm94eSIsCiAgICAgICAgICAgICJlZGl0YWJsZSI6IGZhbHNlLAogICAgICAgICAgICAibmFtZSI6ICJwcm9tZXRoZXVzIiwKICAgICAgICAgICAgIm9yZ0lkIjogMSwKICAgICAgICAgICAgInR5cGUiOiAicHJvbWV0aGV1cyIsCiAgICAgICAgICAgICJ1cmwiOiAiaHR0cDovL3Byb21ldGhldXMtazhzLm1vbml0b3Jpbmcuc3ZjOjkwOTAiLAogICAgICAgICAgICAidmVyc2lvbiI6IDEKICAgICAgICB9CiAgICBdCn0=
+kind: Secret
+metadata:
+  name: grafana-datasources
+  namespace: monitoring
+type: Opaque
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana
+  namespace: monitoring
+---
+{%- raw %}
+apiVersion: v1
+items:
+- apiVersion: v1
```

Comment:
:poop:

Agreed that it's done automatically with the bundler tool, but we are just pushing a bunch of JSON dashboards into ConfigMaps, let's just store the JSON files somewhere and include them in this YAML, instead of having a 7k lines file.

Also, dashboard addition will need to be "standardized" so that Solutions can add theirs as well (maybe we'll need to rework something in the Grafana Pod spec).

Reply:
So, let's leave it as is for now. However, it would be interesting to consider using something like [`kustomize`](https://kustomize.io/) in the future (they have a feature to generate ConfigMaps from files).